### PR TITLE
Picker buttons

### PR DESCRIPTION
**Pull request by ray007**
## Proposed changes
- make clear button in datepicker and timepicker optional
- add cancel button to datepicker (optional)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:

- [x] I have read the **[CONTRIBUTING document](https://github.com/Dogfalo/materialize/blob/master/CONTRIBUTING.md)**.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by DanielRuf** [comment]
It seems like this is a breaking change and changes the old behavior (additional things added by default without setting any of the new options from the outside).

A clear button generally makes sense and improves the UX. Can you describe the cases where it makes sense to make it optional or remove it?

Doesn't the additional cancel button make this a bit crowded? What about a normal `&times;` close button at the top right for the picker modal?

**Comment by ray007** [comment]
Breaking change? I thought I took care to add the options in a way to not change anything from the current behavior unless the new options were used. Well, other then the "today" button moving a bit to the left.

As for the clear button: I have implemented a scheduler control. When editing an event, not specifying a date is not a valid option.

All taken from the google guideline: https://material.io/guidelines/components/pickers.html

File: js/datepicker.js [hunk 1]
```diff
@@ -50,10 +50,13 @@
     // Specify a DOM element to render the calendar in
     container: null,
 
+    clearBtn: true,
```
[the remark this answers is not in the document]
Comment (ray007) [reply js/datepicker.js]: It's enabled by default in order to not change the current behavior: show a clear button.
Would you like a different button configuration mechanism?

**Comment by ray007** [comment]
Looking at the travis log, the test warnings don't seem to be connected to my changes.

**Comment by DanielRuf** [comment]
jwuery 3.3.1 might be the cause, see https://github.com/jquery/jquery/issues/3955

I'll do a local test to verify this.

**Comment by ray007** [comment]
Alternate fix by Alvin Wang works well enough, closing.

**Pull request closed without merging**